### DISCUSSION TITLE
Guild's System - SFL Tribe House 🏡 (Full proposal PDF file attached)

### DISCUSSION THREAD

**Discussion by fishretailer** [sunflower-land/sunflower-land#1960] [Ideas]
**_Tribe House Proposal for Sunflower Land_**
(System related to Guild of Farming Game)
![image](https://user-images.githubusercontent.com/121532185/210192426-ddcd3491-11c3-4bf3-a68a-7a943897b6f0.png)
**Proposal overview :**
![image](https://user-images.githubusercontent.com/121532185/210192467-74f4d489-7fab-4e60-aa32-341fb32dfe4e.png)

**Introduction : We can use Treasure Island to introduce An Ancestor's Artifact (An ancient wooden Tribe Mask)
By discovering that treasure , we were blessed by our ancient ancestor that live in the Isle before us. We learn the power of civilization by building a tribe, a knowledge to work together as a tribe.**
![image](https://user-images.githubusercontent.com/121532185/210192478-60dad661-e322-4a21-8354-9b93a096c930.png)

**Tribe House Definition : A TRIBE HOUSE can contain from 1-30 Bumpkins. Bumpkins can create or join a tribe house by building it in a new section (section like goblin retreat, Hellios...etc)**

**To build a tribe house you need to spend
50 golds + goblin crown + 1000 woods + 100SFL**
![image](https://user-images.githubusercontent.com/121532185/210192502-ad72e3ff-bff3-422f-9cfa-08b15ab18ba1.png)

**Ancestor whisper: _"if you lower down your ego, giving out the treasure of your war achievement, you will be a great Tribe Leader"_
That was the ideas of burning Goblin Crown NFT + Huge Resources (all can be replace by Dev since they will know the economy so well)**

**In exchange : Tribe will receive a Tribe House NFT or TRIBE HOUSE GUILD ID**
![image](https://user-images.githubusercontent.com/121532185/210192509-8b5b9243-3848-4727-910a-58b22509eacf.png)

--------------------------------------------------
**+Economy explain :**
**_(Creating Tribe House = Resource + SFL Burning) -> (Tribe House will have a weekly BUFF, by joining Fire Ritual Activity) -> (Tribe earn more) -> (Upgrading Tribe House = Burning More)_**
![image](https://user-images.githubusercontent.com/121532185/210192520-a00a0551-9cb1-488e-a208-06f7b3012842.png)

--------------------------------------------------
*** Tribe House Activities (we can unlock stage by stage no need to release at once)**

+ Chat with others Bumpkins
+ Weekly fireside ritual ( to have a little time to buff their farming or stuff, so this could help everyone to play in a specific time and can have teambuilding together), this also depend of the tribe level to have this
![image](https://user-images.githubusercontent.com/121532185/210192530-7374d096-5ba5-4810-8350-a554dea80914.png)

+ Visit their Tribe and leaving note (or doing some small task that do not involve to multi-accounts)
+ They can request/help giving resources/crops to other Bumpkins
(By finishing this task u can have different TAG to showcase that your Bumpkin is a enthusiast bumpkin that willing to help)
![image](https://user-images.githubusercontent.com/121532185/210192538-6c7bc9a9-b4d2-4258-a127-a2515063d890.png)

+ Generate new resources
+ Playing team activities to compete with other tribes
(for example level up their tribe house to increase buff or obtain rare NFT for showcase purpose, for example each day each tribe members will receive random tasks, by finishing those we can gain Experience for tribe house, and this is included burning system too)
![image](https://user-images.githubusercontent.com/121532185/210192545-5d01922f-3958-472e-a096-d41e6d374a00.png)

By winning competition they will win some RARE NFT TO PROVE THAT THIS IS THE RANK 1 TRIBE
(will obtain some TRIBE NFT for bumpkin)

+ Highest Tribe can have benefit like GUILD ID instead of LAND ID
![image](https://user-images.githubusercontent.com/121532185/210192569-50a436b4-8d58-46ef-bfea-2a70f01bc30a.png)

--------------------------------------------------
Later Tribe/Lore expand ideas (we can save for Tribe 2.0 or Tribe 3.0 for developers coding)

In TRIBE's area we can expand TRIBE LAND

For example : 1 Tribe Land expand require : 10 golds + 3 mutant crops + nX (*special resource that didn't exist yet)
For example 2 : 1 Tribe Land should require : Burning 1 - 3 Sunflower Land to burn more Lands
![image](https://user-images.githubusercontent.com/121532185/210192586-dcf5c0ae-e8de-49fe-9dbf-c5158680b42e.png)

In exchange : Tribe will have a special Tribe Land that their Tribe Members can Access and doing task to harvest special Resources / Crops / at this point i was thinking some special Utility of chicken Eggs
![image](https://user-images.githubusercontent.com/121532185/210192601-3bcdd235-42d8-4c98-8d8b-d8ccbf41c9c5.png)

Since the data can show that Chicken/Egg is not having as much as utility they should have

and in tribe land we will have Tribe's Plot, Tribe's Resource, and each tribe will have their own rule to how to distributed it to their TRIBE's MEMBER
--------------------------------------------------
Impact of Releasing Tribe House : As a gamer we would love to have interacting with members, as thinking of the propose i think A Tribe/Guild system is something that can have mass adoption for new players (more activities, more time spend, more interact so they can know the game better than playing alone)
![image](https://user-images.githubusercontent.com/121532185/210192612-b525bf28-e236-4ffa-be2e-7116b5ae23e5.png)

Side note : Using Tribe system can help Burning more Resources/Crops/Token and we can make the benefit of the Tribe slowly gain but rewarding player with some cool Bragging NFT by decorating their Tribe Land
[Tribe-House.pdf](https://github.com/sunflower-land/sunflower-land/files/10329906/Tribe-House.pdf)

**Comment by NomiceBinocle** [comment]
brilliant proposal I love the principle

only small downside, only 1 player should need to craft the nft to create the tribe, the others will have to be able to integrate it unconditionally.
This will allow to mix high level and bat level players in the same tribe which would be better to attract new people

otherwise I can't wait to see the evolution of this proposal and even to see it in games

**Reply by fishretailer** [reply]
would love more idea like this to help contribute this project <3

**Reply by cryptosmikey** [reply]
Amazing idea
